lib/sorbet-rails/active_record_rbi_formatter.rb

```diff
@@ -69,16 +71,7 @@ def generate_active_record_relation_rbi
         return_type: "T::Array[T.type_parameter(:U)]",
       )
 
-      class_rbi.create_method(
-        "exists?",
-        parameters: [ Parameter.new("conditions", type: "T.untyped", default: "nil") ],
-        return_type: "T::Boolean",
-      )
-      class_rbi.create_method('any?', return_type: "T::Boolean")
       class_rbi.create_method('empty?', return_type: "T::Boolean")
```

Comment:
`empty?` is actually only available on the relation classes, not on the model itself. All the others (exists?, any?, etc) are all available on both the model and the relation classes.

Reply:
O.o 
Can you add your message as a comment in the code?